fmf/base.py

```diff
@@ -92,10 +133,10 @@ def __init__(self, data, name=None, parent=None):
         if self.parent is None:
             self.inherit()
 
-        log.debug("New tree '{0}' created.".format(self))
+        log.debug(f"New tree '{self}' created.")
```

Comment:
`format()` => f-string change seems to be unrelated to adding type annotations, which adds more noise to the patch. Is it necessary to bundle two distinct changes into a single PR?

Reply:
Hmm, indeed, I should split it to a separate PR, should it be a standalone PR or bundle a few other style changes (though not sure what to add, maybe converting to `pathlib`?)

Reply:
I for one prefer distinct PRs, i.e. one for `format()` vs f-string, another one for `pathlib`, and so on. It's easier for me to review (and revert...) such changes. It's of course possible to create nice, distinct commits focusing one one thing only, and put more of them into a single PR, but that's often a pain when rebasing. For me, one PR for one change wins.